README.md

```diff
@@ -52,6 +52,17 @@ pip3 install -r requirements.txt
 python Cr3dOv3r.py -h
 ```
 
+**+For macOS :**
+```
+git clone https://github.com/D4Vinci/Cr3dOv3r.git
+chmod -R 777 Cr3dOv3r
+cd Cr3dOv3r
+pip install -r requirements.txt
+python Cr3dOv3r.py -h
```

Comment:
How does brew / XCode handles concurrent versions of python ? When you call ```python```, are you sure that it is ```python3``` that is launched, and not ```python2``` ?

Reply:
Running `python` on MacOS uses the system version (2.7 on Sierra). Even though the docs says v3 is required, the script seems to work with v2.7.

Reply:
Updated docs for using Python 3 on MacOS. 
1. Should I remove the reference to v2.7? 
2. Should I include the steps for installing Homebrew and Python3 (currently external link)?

Reply:
I don't know, it depends on @D4Vinci will's to maintain a script on two different versions of python (There are no differences now, could be in the future). His call.
As for the documentation, I personally thinks that an external links is good enough, since you don't want to make an update every time the python or the homebrew community makes a change on how to install their software.